data/apps.ts

```diff
@@ -136,7 +137,14 @@ export const apps: appsList = {
       icon: mammoth,
       url: "https://apps.apple.com/app/mammoth-for-mastodon/id1667573899",
       paid: false
-    }
+    },
+    {
+      released_on: "Mar 27, 2023",
+      name: "Nightfox DAWN",
```

Comment:
In the iOS App Store this is called "DAWN for Mastodon", please update this here to that it matches. Thanks!